src/living/helpers/babylonjs/InteractiveDynamicTexture.ts

```diff
@@ -337,6 +356,48 @@ export class InteractiveDynamicTexture extends BABYLON.DynamicTexture {
     return isDirtyAfterRendering;
   }
 
+  // cannot call it in `render()`, cuz it cost too much.
+  static _parserTransform(transformStr: string): DOMMatrix {
```

Comment:
The CSS property parsing should not be a rendering-related source file, please do this work in CSS-related files.